@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -111,14 +127,49 @@ void main() {
 
     float alpha = opacity;
 
+    float lineEnds = max(vExtrusionCoord.x - vLength,- vExtrusionCoord.x);
+
     #if TILE_CLIP
     tileClip(vPosition.xy, tileSize);
     #endif
 
-    float dist = joinDist(vSegment, vExtrusionCoord) - vLinewidth;
+    float dist = 0.0;
+
+    #if defined(CAPS_NONE)
```

Comment:
Cool stuff! Would add it to the `LinesChunks.ts` file to clean up the main shader code, but pretty neato.

Reply:
Done